server/consumer.go

```diff
@@ -1489,6 +1577,59 @@ func (o *consumer) sendDeleteAdvisoryLocked() {
 	o.sendAdvisory(subj, j)
 }
 
+func (o *consumer) sendPinnedAdvisoryLocked() {
+	group := _EMPTY_
+	if len(o.cfg.PriorityGroups) == 0 {
+		group = o.cfg.PriorityGroups[0]
```

Comment:
If `PriorityGroups` is length 0, then `PriorityGroups[0]` is a out-of-bounds panic.